services/openvpn/config_validator.go

```diff
@@ -121,6 +123,24 @@ func validTLSPresharedKey(config *VPNConfig) error {
 	if len(key) != 512 {
 		return errors.New("invalid key length")
 	}
+
+	var buff = &bytes.Buffer{}
+	fmt.Fprintln(buff, "-----BEGIN OpenVPN Static key V1-----")
+	left := key
+	for {
```

Comment:
can't we do it like this?

```golang
for left := key; len(left) > 64; left = left[64:] {
   fmt.Fprintln(buff, left[0:64])
}
fmt.Fprintln(buff, left)
```

Reply:
Yes we can. Also we can write additional test for this code part. Done